nebula-components/_components.scss

```diff
@@ -7,4 +7,5 @@
 @import './components/navbar';
 @import './components/search';
 @import './components/select';
+@import './components/tab';
```

Comment:
Should be pluralised in this case I think - our React component is `<Tabs />` and they're never gonna be used singularly.